beacon_node/beacon_chain/src/fetch_blobs.rs

```diff
@@ -73,13 +73,23 @@ pub async fn fetch_and_process_engine_blobs<T: BeaconChainTypes>(
         .as_ref()
         .ok_or(FetchEngineBlobError::ExecutionLayerMissing)?;
 
+    inc_counter_by(
+        &metrics::BLOBS_FROM_EL_EXPECTED_TOTAL,
+        num_expected_blobs as u64,
+    );
```

Comment:
You could also change to a histogram here. Still has a total but it's more granular. Plus add a counter for `FetchEngineBlobError::RequestFailed`. Then we can use `ERROR / (BLOBS_FROM_EL_EXPECTED_TOTAL + ERROR)` for the error rate. And know how often we request blobs for different blob counts.

Reply:
Updated, please re-review, thanks!